plugin/core/state.py

```
@@ -26,19 +76,989 @@ def __init__(self):
+            self._lifecycle_store = new_store
+        if removed:
+            try:
+                rev = self._bump_bus_rev("lifecycle")
+                self.bus_change_hub.emit("lifecycle", "del", {"lifecycle_id": lifecycle_id, "rev": rev})
+            except Exception:
+                pass
+        return removed
+    
+    def set_plugin_response(self, request_id: str, response: Dict[str, Any], timeout: float = 10.0) -> None:
+        """
+        设置插件响应（主进程调用）
```

Ruff (0.14.13)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)